language/solana/move-to-solana/Cargo.toml

```diff
@@ -37,19 +38,15 @@ once_cell = "1.10"
 parking_lot = "0.11"
 regex = "1.1.9"
 semver = "1.0.13"
-tempfile = "3.2"
-toml = "0.5.8"
-
-[dev-dependencies]
-datatest-stable = "0.1.1"
 serde = { version = "1.0.124", features = ["derive"] }
 serde_json = "1.0.64"
-similar = "2.1.0"
 solana-bpf-loader-program = { git = "https://github.com/solana-labs/solana", rev = "9731331be4dd4bfccf85b0443e3d652aec30a1f8" }
 solana-program-runtime = { git = "https://github.com/solana-labs/solana", rev = "9731331be4dd4bfccf85b0443e3d652aec30a1f8" }
 solana-sdk = { git = "https://github.com/solana-labs/solana", rev = "9731331be4dd4bfccf85b0443e3d652aec30a1f8" }
 solana_rbpf = "=0.6.0"
-thiserror = "1.0.37"
+tempfile = "3.2"
+
+[dev-dependencies]
```

Comment:
This crate uses previous dev deps as main dependencies (all solana-* crates), because the test runner, that depends on solana-* crates, is part of the move-cli when move-cli is built with solana-backend enabled.